src/TypeResolver.php

```diff
@@ -47,7 +48,8 @@
         'true' => 'phpDocumentor\Reflection\Types\Boolean',
         'self' => 'phpDocumentor\Reflection\Types\Self_',
         '$this' => 'phpDocumentor\Reflection\Types\This',
-        'static' => 'phpDocumentor\Reflection\Types\Static_'
+        'static' => 'phpDocumentor\Reflection\Types\Static_',
+        'iterable' => Iterable_::class,
```

Comment:
I've used `::class` notation here which I realise is inconsistent with the rest of the tests, so I can change this if preferred, just let me know please :)